### DISCUSSION TITLE
[AssetMapper] How to install bootstrap-icons?

### DISCUSSION THREAD

**Discussion by tacman** [symfony/symfony#52583] [Q&A] [answered]
Usually when I want to install a package with AssetMapper, I go to JSDelivr and look for it, e.g. bootstrap-icons

https://www.jsdelivr.com/package/npm/bootstrap-icons

OR bootswatch, also css only

https://www.jsdelivr.com/package/npm/bootswatch

Then I install it using the same name as npm

```bash
in/console importmap:require bootstrap-icons

In JsDelivrEsmResolver.php line 119:
                                                                                                                                           
  Error 404 requiring packages from jsDelivr for "bootstrap-icons". Check your package names. Response: Couldn't find the requested file.  
                                                                                                                                           

In CommonResponseTrait.php line 173:
                                                                                        
  HTTP/2 404  returned for "https://cdn.jsdelivr.net/npm/bootstrap-icons@1.11.1/+esm".  
                                                                                        

importmap:require [--entrypoint] [--path PATH] [--] <packages>...
```

This makes sense, because there's no javascript ESM, it's a bonus that if there's a `style` key in the package.json then the css gets imported as well.  But what about this case, where there's no javascript?  I realize I can add it other ways, but since bootstrap-icons installs with npm, intuitively it feels like it should be installable with AssetMapper.

I imagine there's a way to just get the CSS, but I'm not sure what the format is.

```bash
bin/console importmap:require bootswatch/cerulean/bootstrap.min.css
```

**Comment by stof** [comment]
which version of AssetMapper are you using ?

**Reply by tacman** [reply]
6.4.0-BETA3

**Reply by tacman** [reply]
same for 6.4-RC1

**Comment by tacman** [comment]
@weaverryan any chance AssetMapper can support css-only and packages like this?  It _seems_ like it's just a matter of creating the importmap entry without an associated javascript one.

**Comment by tacman** [comment, marked as answer]
I had the format wrong, I see I can download these as

```bash
bin/console importmap:require bootstrap-icons/font/bootstrap-icons.min.css
bin/console importmap:require bootswatch/cerulean/bootstrap.min.css
```

**Comment by tacman** [comment]
Actually, even with the right installation, bootstrap-icons don't load as expected.  https://github.com/symfony/symfony/issues/52620

**Comment by Ornella85** [comment]
Hi, 
It works with : php bin/console importmap:require bootswatch/dist/cyborg/bootstrap.min.css
Choice your bootswatch theme and : php bin/console importmap:require bootswatch/dist/[yourtheme]/bootstrap.min.css
